pkg/virt-controller/watch/application_test.go

```diff
@@ -142,6 +142,7 @@ var _ = Describe("Application", func() {
 			cdiConfigInformer,
 			config,
 			topology.NewTopologyHinter(&cache.FakeCustomStore{}, &cache.FakeCustomStore{}, nil),
+			nil,
```

Comment:
Consider returning an error if this is `nil`.

Reply:
I've added a dummy anonymous function here instead of a nil.
The error code is explicitly ignored by this test.

The VMI controller will panic if a nil will be passed in production, causing the e2e tests to fail.